code/game/objects/items/devices/scanners/sequence_scanner.dm

```diff
@@ -31,6 +38,31 @@
 	else
 		user.visible_message(span_notice("[user] fails to analyze [target]'s genetic sequence."), span_warning("[target] has no readable genetic sequence!"))
 
+/obj/item/sequence_scanner/attack_secondary(mob/living/target, mob/living/carbon/human/user, max_interact_count=1)
+	add_fingerprint(user)
+	//no scanning if its a husk or DNA-less Species
+	if (!HAS_TRAIT(target, TRAIT_GENELESS) && !HAS_TRAIT(target, TRAIT_BADDNA))
```

Comment:
I'd prefer if you also checked for TRAIT_NO_DNA_COPY

Reply:
Wait, Wouldn't this be a surefire way to test a cling?